content/sdk/10.js/00.ethers/20.v6/01.features.md

```diff
@@ -79,25 +69,29 @@ and paymaster input `0x8c5a3445`:
 }
 ```
 
-## Encoding paymaster params
+::callout{icon="i-heroicons-light-bulb"}
+Everything inside `customData` in `overrides` is related to zksync(L2 gas, etc.).
+::
```

Comment:
This should be a part of the description up at the beginning. This is too far down the page to mention.